modules/auxiliary/gather/enum_dns.rb

```diff
@@ -1,483 +1,420 @@
-      'Name'		   => 'DNS Record Scanner and Enumerator ',
-      'Description'	=> %q{
+      'Name'           => 'DNS Record Scanner and Enumerator',
+      'Description'    => %q{
         This module can be used to gather information about a domain from a
         given DNS server by performing various DNS queries such as zone
         transfers, reverse lookups, SRV record bruteforcing, and other techniques.
-
       },
-      'Author'		=> [ 'Carlos Perez <carlos_perez[at]darkoperator.com>' ],
-      'License'		=> MSF_LICENSE,
-      'References' 	=>
-        [
-          ['CVE', '1999-0532'],
-          ['OSVDB', '492'],
-        ]
-      ))
+      'Author'         => [
+        'Carlos Perez <carlos_perez[at]darkoperator.com>',
+        'Nixawk'
+      ],
+      'License'        => MSF_LICENSE,
+      'References' 	   => [
+        ['CVE', '1999-0532'],
+        ['OSVDB', '492']
+      ]))
 
     register_options(
       [
-        OptString.new('DOMAIN', [ true, "The target domain name"]),
-        OptBool.new('ENUM_AXFR', [ true, 'Initiate a zone transfer against each NS record', true]),
-        OptBool.new('ENUM_TLD', [ true, 'Perform a TLD expansion by replacing the TLD with the IANA TLD list', false]),
-        OptBool.new('ENUM_STD', [ true, 'Enumerate standard record types (A,MX,NS,TXT and SOA)', true]),
-        OptBool.new('ENUM_BRT', [ true, 'Brute force subdomains and hostnames via the supplied wordlist', false]),
-        OptBool.new('ENUM_IP6', [ true, 'Brute force hosts with IPv6 AAAA records',false]),
-        OptBool.new('ENUM_RVL', [ true, 'Reverse lookup a range of IP addresses', false]),
-        OptBool.new('ENUM_SRV', [ true, 'Enumerate the most common SRV records', true]),
-        OptPath.new('WORDLIST', [ false, "Wordlist for domain name bruteforcing", ::File.join(Msf::Config.data_directory, "wordlists", "namelist.txt")]),
-        OptAddress.new('NS', [ false, "Specify the nameserver to use for queries (default is system DNS)" ]),
-        OptAddressRange.new('IPRANGE', [false, "The target address range or CIDR identifier"]),
-        OptBool.new('STOP_WLDCRD', [ true, 'Stops bruteforce enumeration if wildcard resolution is detected', false])
+        OptString.new('DOMAIN', [true, 'The target domain']),
+        OptBool.new('ENUM_AXFR', [true, 'Initiate a zone transfer against each NS record', true]),
+        OptBool.new('ENUM_STD', [true, 'Enumerate standard record types (A,MX,NS,TXT and SOA)', true]),
+        OptBool.new('ENUM_BRT', [true, 'Brute force subdomains and hostnames via the supplied wordlist', false]),
+        OptBool.new('ENUM_TLD', [true, 'Perform a TLD expansion by replacing the TLD with the IANA TLD list', false]),
+        OptBool.new('ENUM_SRV', [true, 'Enumerate the most common SRV records', true]),
+        OptBool.new('STOP_WLDCRD', [true, 'Stops bruteforce enumeration if wildcard resolution is detected', false]),
+        OptAddress.new('NS', [false, 'Nameserver for query']),
```

Comment:
You should retain the part of the original description that indicates that the system DNS servers will be used if none are specified.